addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationPlan.java

```diff
@@ -54,7 +54,7 @@ public class AutomationPlan {
     private Date finished;
 
     private static final Logger LOGGER = LogManager.getLogger(AutomationPlan.class);
-    private static final ObjectMapper YAML_OBJECT_MAPPER;
+    public static final ObjectMapper YAML_OBJECT_MAPPER;
```

Comment:
Better to add method to write an object instead of exposing internal fields and duplicating code which will most likely get out of sync.